specification.rst

```diff
@@ -384,6 +384,9 @@ the item type is "book":
       </citation>
     </style>
 
+The output of a ``<text macro="...">`` does not have delimiters from any
```

Comment:
Thanks for working on this! I think we're almost there. I think we can improve on interspersed. 
Does "The output of a ``<text macro="...">`` does not take delimiters from any
ancestor delimiting element" (and similarly below) capture it? 
"Interspersed" sounds like a random process to my (admittedly non-native speaker) ears

Reply:
I'm lost at this point. Can you guys "resolve" the suggestions or comments that you've addressed, once you have, so that we can tell when ready to merge?

Reply:
From my end this is the only remaining unresolved issue. I've left your proposed change unresolved (since it's yours)